From 2793c9305a1092c03d04a5fe0fa3a56811039447 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Tue, 1 Sep 2015 11:02:07 -0400
Subject: [PATCH] Specify bootloader directory in the test case for 1152550.

Because otherwise it blows up when built in mock without /boot mounted.

Related: rhbz#1152550

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 test.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test.sh b/test.sh
index cd2d870..ba466a5 100755
--- a/test.sh
+++ b/test.sh
@@ -547,7 +547,7 @@ if [ "$testgrub2" == "y" ]; then
     # has this being emitted as: set foo="bar=1,2"=1,2"
     # which is wrong.
     grub2Test grub2.16 add/g2-1.16 \
-        --add-kernel=/boot/vmlinuz-foo \
+        --boot-filesystem=/boot --add-kernel=/boot/vmlinuz-foo \
         --copy-default --title 'Red Hat Enterprise Linux Server' \
         --args=root=/dev/mapper/foo--
 
-- 
2.4.3

